### PR TITLE
Fixes for airtime and timeouts

### PR DESCRIPTION
**Pull request by ericnewcomer**
* Support for supra-national currencies (USD, EUR)
* Cleaner configuration based implementation of rulesets with prescribed rules
* Temorarily disable webhook routing rules
* Temporarily disable airtime transfers
* Much more dry test_controllers
* Ensures rules are permitted for a given ruleset type
* Use Success instead of Completed for airtime

### CODE REVIEW

**Comment by ericnewcomer** [comment]
Can I get a what what? @nicpottier 

The failing tests are for airtime because the permissions are temporarily disabled.

**Comment by norkans7** [comment]
Yep

File: static/coffee/flows/controllers.coffee [hunk 1]
```diff
@@ -884,12 +883,19 @@ NodeEditorController = ($rootScope, $scope, $modalInstance, $timeout, $log, Flow
...
     {value:120, text:'2 hours'},
     {value:180, text:'3 hours'},
-    {value:360, text:'6 hours'}
+    {value:360, text:'6 hours'},
+    {value:720, text:'12 hours'},
+    {value:1080, text:'18 hours'},
+    {value:1440, text:'1 day'},
+    {value:2880, text:'2 days'},
+    {value:4320, text:'3 days'},
+    {value:10080, text:'1 week'},
```
Comment (nicpottier) [inline static/coffee/flows/controllers.coffee]: Nice

**Comment by nicpottier** [comment]
Doit!

**Pull request merged**